client/src/components/pages/CreateBusiness.jsx

```diff
@@ -21,3 +21,26 @@ const CreateBusiness = () => (
 );
 
 export default CreateBusiness;
+// import React from 'react';
```

Comment:
HiAnn, seriously I believe that you write good codes, really great implementation, but like I earlier said, commented out codes make your work messy and unprofessional. Please clean them up to keep you at the top of your game. great work.

Reply:
Thanks for the feedback, I will incorporate all you have said